bundles/CoreBundle/src/Migrations/Version20230112203416.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
Like I wrote in my comment before, we do not need the Migration at all. You can remove the whole file.